### ISSUE TITLE
favicon not working

### ISSUE THREAD

**Issue by alrek-consulting** [PrismarineJS/node-minecraft-protocol#397]
I tried to start a server with these options:

```
mc.createServer({
    "online-mode": true,
    encryption: true, 
    host: "0.0.0.0", 
    port: 25565,
    "max-players": 20,
    version: 1.9,
    motd: "A Minecraft Server",
    favicon: "iVBORw0KGgoAAAAN ... AAAASUVORK5CYII="
});

```
The server launches just fine, and I can connect, however, the server icon does not appear. Am I doing something wrong?

**Comment by rom1504** [comment]
looks like it's not handled in the option https://github.com/PrismarineJS/node-minecraft-protocol/blob/3b50ea9eb7ee1662396e0ee59ddfb6617d062230/src/createServer.js#L26

feel free to do a PR to add something like `server.favicon=options.favicon;` at that line and make sure it works

**Comment by alrek-consulting** [comment]
Fixed in commit #9831bdc4c203aeed751ded5ea03551a90bb5ae41
